src/components/Main/BookList.module.scss

```diff
@@ -0,0 +1,33 @@
+@import "../Variables.scss";
+
+.book-list-ppal {
+  height: 100%;
+  width: 100%;
+  background-color: map-get($colors, superLightBrown );
+  position: relative;
+}
+.books-list {
+  background-color: map-get($colors, superLightBrown);
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  grid-column-gap: 2rem;
+  grid-row-gap: 2rem;
+  margin: 2rem;
+}
+@media (min-width: map-get($breakPoints, small)) and (max-width: (map-get($breakPoints, medium) - 1)) {
```

Comment:
puedes crear mixins que te ayuden a tener mayor orden en cuanto a los media queries

Reply:
ej: https://medium.com/codeartisan/breakpoints-and-media-queries-in-scss-46e8f551e2f2